cli/azd/cmd/down.go

```diff
@@ -97,30 +98,34 @@ func newDownAction(
 }
 
 func (a *downAction) Run(ctx context.Context) (*actions.ActionResult, error) {
-	infraManager, err := provisioning.NewManager(
-		ctx,
-		a.env,
-		a.projectConfig.Path,
-		a.projectConfig.Infra,
-		a.console.IsUnformatted(),
-		a.azCli,
-		a.console,
-		a.commandRunner,
-		a.accountManager,
-		a.userProfileService,
-		a.subResolver,
-		a.alphaFeatureManager,
-	)
-
+	// silent manager for running Plan()
+	infraManager, err := createProvisioningManager(ctx, a, &project.MutedConsole{ParentConsole: a.console})
 	if err != nil {
 		return nil, fmt.Errorf("creating provisioning manager: %w", err)
 	}
 
+	// Command title
+	a.console.MessageUxItem(ctx, &ux.MessageTitle{
+		Title:     "Deleting all resources and deployed code on Azure (azd down)",
+		TitleNote: "Local application code is not deleted when running 'azd down'.",
```

Comment:
@savannahostrowski @Austinauth Do you think we could make these more concise for better UX?

I was thinking along the lines of:

"Delete Azure resources (azd down)"
"Cleaning up provisioned infrastructure and services in Azure. Local application files will be kept."

I'm still not sure about the mention about "application files" being kept. Most computer users may expect programs to not remove their files unless explicitly prompted so (products that violate this probably don't end up living long), and maybe the help text/description suffices. It feels close to "Windows updates" reassurance of "your files are exactly where you kept them", which always reassures me less as a user that Windows needs to bring that up (and because it has happened, and still happens today).

Reply:
@weikanglim, Savannah and I discussed this previously when we made the change.

The reason for this title and description is because we observed that some users failed to understand where the code was deleted vs were it was kept (including myself on first use).

I am okay consolidating the primary title back to "Deleting Azure resources (azd down)", however this puts more pressure on the title note to be more descriptive .The proposed title not change does not feel any more clear to me than what was initially proposed.